### ISSUE TITLE
Provide way to customise key bindings

### ISSUE THREAD

**Issue by asfaltboy** [noahcoad/open-url#58]
Ctrl+U is mapped to a bunch of things in my OSX env, and I'd like to be able to use open-url and Ctrl+U without needing to check it out as source and comment out the bindings ...

One way this can be done is by providing a conditional context based on a settings such as modifying [the current key binding to](https://github.com/noahcoad/open-url/blob/f801c7f92f09c7f1deafaec9a5a120153b3fdfbb/Default%20(OSX).sublime-keymap) (and adding the appropriate setting):

```json
[
    {
        "keys": ["ctrl+u"],
        "command": "open_url",
        "context": [
            { "key": "setting.open_url.enable_default_shortcuts", "operator": "equal", "operand": true }
        ]
    }
]
```

**Comment by kylebebak** [comment]
Hey @asfaltboy , thanks for creating this issue. I didn't realize this was an issue until now, and agree this is not ideal. That said, on my machine running macOS 14.0 and ST 4152 I can't seem to repro ST key bindings shadowing key bindings from other programs. Rather, key bindings from other programs (e.g. from iTerm2 and Moom) shadow those from ST

I've modified your solution so it keeps backwards compatibility and doesn't require users to enable shortcuts explicitly by editing `Preferences.sublime-settings`

```json
[
  {
    "keys": [
      "ctrl+u"
    ],
    "command": "open_url",
    "context": [
      {
        "key": "setting.open_url.disable_default_key_bindings",
        "operator": "not_equal",
        "operand": true
      }
    ]
  }
]
```

In other words, if you add `"open_url.disable_default_key_bindings": true` to `Preferences.sublime-settings` the default key bindings will be disabled. On another note, it would be nice if `context` could reference plugin settings. I found a few places, e.g. here, [saying it can't](https://forum.sublimetext.com/t/keymap-context-plugin-setting/2845)

**Comment by kylebebak** [comment]
Fixed in https://github.com/noahcoad/open-url/releases/tag/2.8.1
